team-app/src/card.css

```diff
@@ -20,8 +20,9 @@
 }
 
 .id {
-    background-color: yellow;
+    background-color: rgb(255, 255, 93);
```

Comment:
Don't name CSS class selectors as id, class, h1, body, etc. Name them according to their use cases.

Reply:
Updated.